Library/VerifyGateway.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Xml;
+
+namespace Recurly
+{
+    class VerifyGateway : RecurlyEntity
```

Comment:
Would `VerifyBillingInfo` be a more accurate name?

Reply:
probably.